bahmni-emr/scripts/openmrs.service

```diff
@@ -7,7 +7,10 @@ After=network.service
 Type=forking
 KillMode=none
 GuessMainPID=no
+RuntimeDirectory=openmrs
```

Comment:
@angshu where do we use **RuntimeDirectory** and **LogsDirectory** ??

Reply:
RuntimeDirectory and LogsDirectory are systemd unit files directory. check [link](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#). they are created under /var/run/ and /var/log/ respectively. These are required for systemd to create the directories. especially for /var/run/[service-name] - can not be created by bahmni:bahmni. bahmni user thats associated with the process does not have permission to create a directory (or any symlink) under /var/run/